pkg/activator/throttler.go

```diff
@@ -0,0 +1,173 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package activator
+
+import (
+	"github.com/knative/serving/pkg/queue"
+	"go.uber.org/zap"
+	"sync"
+	"github.com/knative/serving/pkg/client/informers/externalversions/serving/v1alpha1"
+	"k8s.io/client-go/tools/cache"
+	v1alpha12 "github.com/knative/serving/pkg/apis/serving/v1alpha1"
+	"errors"
+	corev1 "k8s.io/api/core/v1"
+	"github.com/knative/serving/pkg/reconciler"
+	"fmt"
+)
+
+const capacityUpdateFailure = "updating capacity failed"
+
+// NewThrottler creates a new Throttler
+func NewThrottler(breakerParams queue.BreakerParams, revisionInformer v1alpha1.RevisionInformer, logger *zap.SugaredLogger) *Throttler {
+	breakers := make(map[RevisionID]queue.Breaker)
+	return &Throttler{breakers: breakers, breakerParams: breakerParams, revisionInformer: revisionInformer, logger: logger}
+}
+
+// Throttler allows to update the max concurrency of registered Breakers
```

Comment:
...allows updating the...? But even so I am not sure I really understand the comment, so perhaps extend it?